package proto_ff;

import "yd_fieldoptions.proto";


message E_DecorateDecorateAttributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_DecorateDecorateActiveattributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_DecorateDecorate
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "装饰ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "装饰名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "外观类型"];
	optional int32 m_quality = 4[(yd_fieldoptions.field_cname) = "眼芒品质"];
	optional string m_professionlimit = 5[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 m_activationitem = 6[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 m_activationnum = 7[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 m_starid = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string m_starnum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 m_starup = 10[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 m_starber = 11[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated E_DecorateDecorateAttributeDesc m_attribute = 12[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
	repeated E_DecorateDecorateActiveattributeDesc m_activeattribute = 13[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_DecorateDecorate
{
	repeated E_DecorateDecorate E_DecorateDecorate_List = 1[(yd_fieldoptions.field_arysize)=40];
}
